docs/reference/search-application/apis/search-application-search.asciidoc

```diff
@@ -8,8 +8,9 @@ beta::[]
 <titleabbrev>Search Application Search</titleabbrev>
 ++++
 
-Given specified query parameters, creates an Elasticsearch query to run. Any unspecified template parameters will be
-assigned their default values if applicable.
+Given specified query parameters, generates and executes an Elasticsearch query using the search template associated
```

Comment:
Some of the same boilerplate suggestions from the previous doc also apply here, we should make sure they're consistent 👍